CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+project(Arbiter C CXX)
+
+cmake_minimum_required(VERSION 3.0)
```

Comment:
It's okay to install things from `apt`—most of the Travis toolchain is pretty old (e.g., C++ compilers only supporting C++11).